ezy-client.js

```diff
@@ -6,13 +6,26 @@ import Socket from './ezy-sockets'
 import EzySetup from './ezy-setup'
 import EzyEventMessageHandler from './ezy-event-message-handler'
 
+/**
+ * Wrapper for JS built-in WebSocket to communicate with websocket server
+ */
 class EzyConnector {
     constructor() {
         this.ws = null;
         this.destroyed = false;
         this.disconnectReason = null;
     }
 
+    /**
+     * Create connection to a websocket server
+     * Websocket events:
+     *      - onerror: fire when failing to connect
+     *      - onopen: fire if succeed to connect
```

Comment:
Nên là: ```fire when connect to server successfully?```

Reply:
Em nghĩ `fire when connection to server is successful` và `fire when connection to server fails` nó đồng bộ và đúng ngữ pháp hơn